### DISCUSSION TITLE
Feasibility Inquiry: Indoor Drone Show with Crazyflie Drones and Lighthouse Positioning System

### DISCUSSION THREAD

**Discussion by kaleez20** [bitcraze/discussions#1118] [Q&A]
Hello,

We are planning to conduct an indoor drone show with Crazyflie drones (50 drones). Is it possible to fly within a 10x10x8 meter area using the Lighthouse position system with more than four base stations? 

Thank you.

**Comment by knmcguire** [comment]
Hi! Thanks for reaching out for this.

So, the basestations V2 have a reported range of 6 meters (see [some specifications](https://www.bitcraze.io/documentation/system/positioning/ligthouse-positioning-system/) here), which will be a bit tricky with only 4 if you place them in the ceiling. We need to double check if it can possibly reach further than that, but if not then there should be some basestations on the side of the flight area.

The more than 4 basestation possibilities, which means that it can handle the signals of 4 basestations at once, is now currently only possible with the prototype lighthouse v2 deck that we showed in the [christmas holiday video](https://www.bitcraze.io/2023/12/merry-christmas-4/), so we hope to make another blogpost with more details. But this does mean that it will still take a while before we release this product this year so there are no dates for that yet. 

Let us discuss this internally and we will come back with more details if we have any

**Reply by kaleez20** [reply]
Thank you for the  information. I look forward to more details on this. Your assistance is much appreciated.

**Reply by knmcguire** [reply]
Hi! So we think that 10 x 10 x 8 will be just too large for just 4 base-stations, but unfortunately this is not something we can test in our own lab. Here we have flown successfully with 9x9x3 meters and we do think that this can be larger than that, if you can consider te 150 degrees FOV and 6 meter range of the basestations, and that the crazyflie can fly with seeing only one basestation at the time, but 10x10x8 might be pushing it.

**Reply by kaleez20** [reply]
Thank you for your information. So, currently, you do not recommend using more than 4 base stations, right?

**Reply by knmcguire** [reply]
Currently it is not possible for the crazyflie to handle more than 4 basestations at the time, unless there is perhaps enough spread that it will only see a couple of the total set. From experience, more than 4 base stations in the same room is very challenging and we do not recommend it. Only once the lighthouse deck v2 is out would this be possible.

**Reply by kaleez20** [reply]
Thank you for your detailed explanation.

**Reply by ashishkumar2025** [reply]
Hello @knmcguire,

I want to ask, if we use 4 Lighthouse base stations, can we achieve a coverage area of 9x9x3 meters? If so, how many Crazyflie 2.1 drones can we connect? I am planning to conduct an indoor drone show with Crazyflie drones and Skybrush software.

Thank you.

**Reply by knmcguire** [reply]
Yes I do believe that is feasible. We have the exact same setup in our lab, although our walls are at the 9 meters so we can't say for sure that is where the coverage truly ends or not.

In terms of how many Crazyflies can connect to the lighthouse system, as many as can receive the sweeps from at least one basestation. The position estimation is done fully onboard so there is no limit there. There is a limit on how many can fly in the same space (prop downwash), and how many can be connected at the same time, but that is depended on the software implementation of the communication. So for that specifically, you'll need to ask the skybrush folks

If you have any more questions I do advise to create a new discussion thread. It's better than responding on older threads like these (but you can always refer to it ofcourse)

**Reply by ashishkumar2025** [reply]
Okay, got it. Thanks for providing the information.
